README.md

```diff
@@ -1,19 +1,19 @@
 # sesci
 
-Jenkins jobs configs can be found in `jenkins/jjb` for jenkins job builder or `jenkins/job-dsl` for Jenkins Job DSL plugin.
+Jenkins jobs configs can be found in `jjb/` for jenkins job builder or `jenkins/job-dsl` for Jenkins Job DSL plugin.
```

Comment:
the '/' is extra here, could you please remove it?

Reply:
Done